gui/Dockerfile.ng-base

```diff
@@ -7,19 +7,17 @@ RUN apt-get -qq update && \
       git \
       sudo \
       vim \
-      curl \
-      npm && \
-    curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash - && \
-    apt-get -qq update && \
+      npm \
+      wget \
+      curl && \
+    curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - && \
     apt-get -qq install nodejs && \
-    ln -s /usr/local/bin/node /usr/local/bin/nodejs && \
     apt-get -qq autoremove && \
     apt-get -qq autoclean && \
     apt-get -qq clean all && \
     rm -rf /var/cache/apk/* /tmp/* /var/lib/apt/lists/*
 
 WORKDIR /root
 
-# Install Grunt + Bower + Protractor
-RUN npm install -g bower grunt-cli && \
-    npm cache clean
+# Install Grunt + Gulp
+RUN npm install -g grunt-cli gulp-cli
```

Comment:
Reworked base image:

- Change base OS from Debian Jessie to Ubuntu Xenial
- Update NodeJS from 6.x to 8.x (LTS)
- Removed Bower from base image